lib/core/versions/latest/DocumentComposer.ts

```diff
@@ -91,13 +105,27 @@ export default class DocumentComposer {
     const didResolutionResult: any = {
       '@context': 'https://www.w3.org/ns/did-resolution/v1',
```

Comment:
https://w3id.org/did-resolution/v1 would be better than a URL that is 404 :)

Reply:
OH I didn't even check it, thanks for catching it!